.github/workflows/build.yml

```diff
@@ -13,8 +13,10 @@ on:
 
 jobs:
   create-runner:
-    uses: ./.github/workflows/create_runner.yml
-    secrets: inherit
+    uses: gitpod-io/gce-github-runner/.github/workflows/create-vm.yml@secrets
```

Comment:
should we update `@secrets` to `@main` before merging (which I guess means merge the PR for the `secrets` branch?). Think otherwise this will break once the secrets branch gets removed due to the PR merging

Reply:
I would prefer to that in a different PR. I am using the same thing for dedicated.

> Think otherwise this will break once the secrets branch gets removed due to the PR merging

Good point. That said, that feature is disabled in the gce-github-runner repository.